packages/dynamite/dynamite/lib/src/builder/ofs_builder.dart

```diff
@@ -8,13 +9,72 @@ import 'package:dynamite/src/models/open_api.dart';
 import 'package:dynamite/src/models/schema.dart';
 import 'package:dynamite/src/type_result/type_result.dart';
 
+TypeResult resolveAllOf(
+  final OpenAPI spec,
+  final State state,
+  final String identifier,
+  final Schema schema, {
+  final bool nullable = false,
+}) {
+  final result = TypeResultObject(
+    '${state.classPrefix}$identifier',
+    nullable: nullable,
+  );
+
+  if (state.resolvedTypes.add(result)) {
+    final interfaces = <TypeResultObject>{};
+
+    for (final s in schema.allOf!) {
+      final TypeResultObject interfaceClass;
+      if (s.ref != null) {
+        final object = resolveType(
+          spec,
+          state,
+          identifier,
+          s,
+          nullable: nullable,
+        );
+
+        if (object is! TypeResultObject) {
+          throw StateError('allOf does only allow objects. Please change $identifier');
+        }
```

Comment:
Why not? I think from the spec sind this is totally allowed?

Reply:
> OpenAPI lets you combine and extend model definitions using the allOf keyword. **allOf takes an array of object definitions** that are used for independent validation but together compose a single object.

https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/#allof

Reply:
Ah ffs I oversaw the `!` and didn't properly read the comment... Sorry for the noise